docs/core/rid-catalog.md

```diff
@@ -112,6 +112,7 @@ Only common values are listed. For the latest and complete version, see the [run
   - `linux-x64` (Most desktop distributions like CentOS, Debian, Fedora, Ubuntu, and derivatives)
   - `linux-musl-x64` (Lightweight distributions using [musl](https://wiki.musl-libc.org/projects-using-musl.html) like Alpine Linux)
   - `linux-arm` (Linux distributions running on ARM like Raspberry Pi)
+  - `linux-arm64` (Linux distributions running on 64-bit ARM like Raspberry Pi Model 4)
```

Comment:
```suggestion
  - `linux-arm` (Linux distributions running on ARM like Raspbian on Raspberry Pi Model 2+)
  - `linux-arm64` (Linux distributions running on 64-bit ARM like Ubuntu Server 64-bit on Raspberry Pi Model 3+)
```

The default Raspbian OS for Raspberry Pi Model 4 is still 32-bit. I think it is important to include the OS here as well to avoid confusion.